index.html

```diff
@@ -5,21 +5,26 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Awesome books</title>
+  <link rel="stylesheet" href="css/book.css">
 </head>
 <body>
   <section>
-    <h1>Book List</h1>
-    <hr>
-    <div id="bookShelf">
-    </div>
+    <h1>All Awesome Books</h1>
+    <div class="bookShelf" id="bookShelf"></div>
   </section>
-
-  <form class="form" id="form">
-    <input id="title" placeholder="Title" type="text" aria-label="bookName" required/>
-    <input id="author" placeholder="Author" type="text" aria-label="authorName" required/>
+  <hr>
+  <section class="main">
+    <h2>Add A New Book</h2>
+    <form class="form" id="form">
+      <input id="title" placeholder="Title" type="text" aria-label="bookName" required/>
+      <br>
+      <input id="author" placeholder="Author" type="text" aria-label="authorName" required/>
+      <br>
+    </form>
     <button class="addButton" id="addButton" type="submit">Add</button>
-  </form>
-
-<script src="./js/script.js"></script>
+  </section>
+  
+<!-- <script src="./js/script.js"></script> -->
```

Comment:
- [ ] Kindly remove commented code as they make your code look messy and hard to read